Gemfile

```diff
@@ -3,19 +3,23 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.3.0'
+
+# NEOVIM IDE / REMEMBER TO UPDATE MASON
+gem 'neovim'
+gem 'solargraph'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem 'rails', '~> 7.0.8'
+gem 'rails', '~> 7.1.2'
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem 'sprockets-rails'
 
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.4'
+# Use pg as the database for Active Record
+gem 'pg', '~> 1.1'
```

Comment:
Changing from sqlite to postgresql was necessary to start preparing for the future. Creating a Docker app with services and in the foreseeable future, to get my app deployed are things that will require Postgresql.